### DISCUSSION TITLE
Change root default password

### DISCUSSION THREAD

**Discussion by porscheme** [vesoft-inc/nebula#5275] [Q&A]
@wey-gu 
Can we change root default password? If yes, steps please.
Thanks

**Comment by QingZ11** [comment]
You can refer to the following documents [https://docs.nebula-graph.io/3.3.0/7.data-security/1.authentication/2.management-user/#change_password](https://docs.nebula-graph.io/3.3.0/7.data-security/1.authentication/2.management-user/#change_password).

as below:

![image](https://user-images.githubusercontent.com/38887077/216265739-4830181c-1ad5-44e5-a9a5-4e07d92d75c9.png)
